lib/Loop/Driver.php

```diff
@@ -572,6 +572,13 @@ protected function error(\Throwable $exception) {
         ($this->errorHandler)($exception);
     }
 
+    /**
+     * Get the current loop time in milliseconds.
+     *
+     * @return int
+     */
+    abstract public function now(): int;
```

Comment:
This is a BC break.

Reply:
Ugh… I suppose so. We could add an `ExtendedDriver`, but I'd rather wait for v3 then.

Reply:
What about adding the method in the implementation classes but not in the interface?

Reply:
@staabm Then it can't be used, because the interface doesn't cover it.

Reply:
Users which dont need to abstract the impl and use it at app level could use it.

Inside amp you couldn‘t

Reply:
The function is mostly useful in libraries, not in applications, so...